### PR TITLE
Updated api.specification.md to adhere to gatsby style guide

### PR DESCRIPTION
**Pull request by adibfirman**
## Description
Avoid pronoun "we" and replace those with "you"

## Related issues
in here #18284

### CODE REVIEW

**Comment by adibfirman** [comment]
@marcysutton Hey there , so this is my very first pull request . So i would like to know if this is fine or let me know if any changes are required. Hope that's fine.

File: docs/docs/api-specification.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ load site config -> load plugins -> source nodes -> transform nodes -> create
 graphql schema -> create pages -> compile component queries -> run queries ->
 fin
 
-Once the initial bootstrap is finished, we start `webpack-dev-server` and an express server for serving files for the development server, and for a production build, we start building the CSS then JavaScript then HTML with webpack.
+Once the initial bootstrap is finished a `webpack-dev-server` will be started, and an express server for serving files for the development server, and for a production build, first is starting to build a CSS then JavaScript then HTML with webpack.
```
Comment (marcysutton) [inline docs/docs/api-specification.md]: How's this? It makes the sentence a bit more active and adds some detail.
```suggestion
Once the initial bootstrap is finished, a `webpack-dev-server` and express server are started for serving files for the development workflow with live updates. For a production build, Gatsby skips the development server and instead builds the CSS, then JavaScript, then static HTML with webpack.
```

Comment (adibfirman) [reply docs/docs/api-specification.md]: thanks for the suggestion, i had commited that :pray:

**Comment by marcysutton** [comment]
Super cool, thanks for the PR!

**Pull request merged**